Dockerfile

```
@@ -11,5 +11,21 @@ RUN make install
 RUN make ui
 RUN make deps
 
+RUN mkdir /stage && \
+    cp -R /go/bin \
+       /go/src/app/templates \
+       /go/src/app/static \
+       /go/src/app/schema.sql \
+       /go/src/app/pages \
+       /go/src/app/keys \
+      /stage && ls -l /stage
+
+FROM golang:1.11.2-alpine3.8
```

```suggestion
FROM scratch
```
Can't the final image be based on scratch or alpine? `go` is probably not needed after building.

---

Yeah, I didn't understand Go doesn't need anything else at runtime; my ignorance. Updated to use alpine:3.8.